### DISCUSSION TITLE
Lab: Descriptive Analysis I

### DISCUSSION THREAD

**Discussion by mbcann01** [brad-cannell/intro.r.public#29] [Labs]
Please feel free to post questions about this lab below. You are also welcome to leave comments below about the usefulness of this lab. Which parts were helpful? What could I do to improve it? What is still unclear?

**Comment by rabdelkhaleq** [comment]
Hi Dr. Cannell! 

I think the link for the Quatro Files R4Epi chapter you have in the quiz instructions is broken...it leads to an error/not found page. 

Current link: https://www.r4epi.com/quarto-files
Correct link (I believe): https://www.r4epi.com/chapters/quarto_files/quarto_files

**Reply by mbcann01** [reply]
Hi @rabdelkhaleq !
Great catch! You were absolutely right. The link should be fixed now.
Thank you!
Brad
